spec/lib/rex/post/meterpreter/ui/console/command_dispatcher/core_spec.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'rex/post/meterpreter/ui/console/command_dispatcher/core'
+require 'lib/msf/ui/console/command_dispatcher/session_spec'
+
+RSpec.describe Rex::Post::Meterpreter::Ui::Console::CommandDispatcher::Core do
+  FactoryBot.define do
```

Comment:
I don't think I've ever seen folk implement factorybot for normal poro domain models - I'm not sure if that would be considered a smell or not 🤔 Even if there is a [thoughtbot article](https://thoughtbot.com/blog/tips-for-using-factory-bot-without-an-orm) on it 😄 

That aside, in isolation this looks like over-engineering as it's only used in this single file from what I can see

Or if the long-term plan is for other tests to use this, then the definitions probably don't belong in this file - they should probably be in the top level support folder? 👀

Reply:
already changed it, realised it wasn't right and meant to swizzle it around before I made the PR but forgot